namespace

Rick-Anderson opened the pull request:
If for some reason your targeted class is in the same project but not in the base namespace you will need…

Rick-Anderson commented:
@ryanbrandenburg  please review - exactly what you suggested.
Do we need to add a section on how to use loc from other projects?

ryanbrandenburg commented:
Looks like a :shipit: to me. We shouldn't have an entire section for cross-project loc because it's not supported out of the box, but I think we should add something explicitly saying cross-project loc isn't supported out-of-box yet, and once we move forward on aspnet/Localization/pull/266 we'll have an example of how to do it yourself to point them at.

Rick-Anderson commented:
I updated the doc with a note that x-prod loc isn't currently supported and updated the issue tracking the loc doc so I can add that section when https://github.com/aspnet/Localization/pull/266 is resolved.
